src/course_bot/presentation.clj

```diff
@@ -127,8 +133,14 @@
     :soon-talk-help "Что произойдет в ближайшее время"
     :schedule-talk "Выберите день презентации"
     :feedback-talk "Отправить отзыв для отчета"
-    :drop-talk-2 "для учителя, отбросить '%s' для конкретного ученика (%s)"
-    :all-scheduled-descriptions-dump-talk "дамп всех запланированных описаний (только для администратора)"}}})
+    :drop-talk-2 "Для учителя, отбросить '%s' для конкретного ученика (%s)"
+    :all-scheduled-descriptions-dump-talk "Дамп всех запланированных описаний (только для администратора)"
+    :approved-presentations-1 "Одобренные доклады в '%s':\n"
+    :submitted-presentations-1 "Отправленные доклады в '%s':\n"
+    :scheduled-decision "ЗАПЛАНИРОВАН"
+    :on-review-decision "НА РАССМОТРЕНИИ"
+    :approved-decision "ОДОБРЕНО"
+    :rejected-decision "ОТКЛОНЕНО"}}})
```

Comment:
Sorry, but here in English we have the same length of all statuses. It will help during the review process. So, we need the same in Russian, or keep English texts here.